### PR TITLE
Added automateAliases

### PR DESCRIPTION
**Pull request by ShayBox**
I added an automateAliases that adds the filename to the mod's aliases if it has an aliases array (command), and the length is 0 and the client option is enabled.

This still allowes you to keep that option as false if you don't like it.
This still allowes you to manually set an array per command and it wont add the filename to it.

### CODE REVIEW

File: src/struct/AkairoHandler.js [hunk 1]
```diff
@@ -113,6 +113,11 @@ class AkairoHandler extends EventEmitter {
             mod.category = dirs[dirs.length - 1];
         }
 
+        if (mod.aliases && mod.aliases.length === 0 && this.client.akairoOptions.automateAliases) {
```
Comment (1Computer1) [inline src/struct/AkairoHandler.js]: AkairoHandler is for all module types. You want to move this to CommandHandler.

Comment (ShayBox) [reply src/struct/AkairoHandler.js]: Ah, okay, I'll fix that

**Comment by ShayBox** [comment]
I also didn't know if I should have make it default to the commands id or file name, I went file name because automateCategories uses foldername, let me know if it would be better to use command id.

**Comment by Reseqo64** [comment]
command id. because we more often use command id as alias than the name of the js

**Comment by ShayBox** [comment]
Alright, I'll change that

**Comment by ShayBox** [comment]
Aparently I had written id in the ClientOption name

**Comment by ShayBox** [comment]
I was thinking about if you wanted to actually have aliases as an empty array, you would have to either turn this off, or set an arbitrary character, would it be a good idea to have a command option that allows you to turn this off per command ex ```automateAliases: false``` in the command options, it defaults to true but is only used if the client option is true (which defaults to false) 🤔 let me know what you think.

**Comment by ShayBox** [comment]
This allows you to disable it from adding the id to aliases per command.

Changed behavior to always add id to aliases if command option is true.

I did this because the old behavior would interfear if you wanted
aliases with other strings but not the id, the only solution was to
disable the client option entirely.

**Comment by ShayBox** [comment]
I'm just going to close this, I don't think it would be merged and it would need to be rewritten with the new changes anyway.

**Pull request closed without merging**